boost/teamcity_boost.cpp

```diff
@@ -97,6 +97,8 @@ struct TeamcityFormatterRegistrar {
 
 BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar);
 
+void TeamcityGlobalFixture(){}
```

Comment:
Don't be intimidated by the complex implementation here :) But that's what makes linker not to throw away entire ` boost/teamcity_boost.cpp `